app/views/breakdowns/MatchBreakdown.js

```diff
@@ -36,4 +36,22 @@ export default class MatchBreakdown extends React.Component {
       <Image source={images.arrows.down} />
     );
   }
+
+  nullHatchPanelImage() {
```

Comment:
Can probably be in the 2019-specific match breakdown, yeah?

Reply:
Good point, I'll get that moved.